packages/server/graphql/private/queries/helpers/authCountByDomain.ts

```diff
@@ -19,13 +19,15 @@ const authCountByDomain = async (
         `SELECT count(*)::float as "total", "domain" from "User"
          WHERE (NOT $1 OR "inactive" = FALSE)
          AND "${filterField}" >= $2
+         AND domain IS NOT NULL
```

Comment:
This basically excludes removed users which is a good thing anyways

Reply:
yep! there are also a handful of users who were created where their email doesn't even include an `@` 🤔. I thought we had a regex protecting against that, so they got tricky.